module/attribute/src/Application/Controller/Api/Attribute/AttributeValidationAction.php

```diff
@@ -94,8 +96,17 @@ public function changeDraftAttribute(
     ): Response {
         $value = $request->request->get('value');
         $value = $value === '' ? null : $value;
+        $aggregateId = null;
+        try {
+            if ($request->query->has('aggregateId')) {
+                $aggregateIdValue = $request->query->get('aggregateId');
+                    $aggregateId =  new AggregateId($aggregateIdValue);
+            }
+            $constraint = $this->provider->provide($attribute, $aggregateId);
+        } catch (\InvalidArgumentException $e) {
+            throw new BadRequestHttpException($e->getMessage());
+        }
```

Comment:
```suggestion
        if ($request->query->has('aggregateId')) {
            $aggregateIdValue = $request->query->get('aggregateId');
            try {
                $aggregateId = new AggregateId($aggregateIdValue);
            } catch (\InvalidArgumentException $e) {
                throw new BadRequestHttpException($e->getMessage());
            }
        }
        $constraint = $this->provider->provide($attribute, $aggregateId);
```
try catch should only be where intended